components/filter/filter-dimension-set-empty-state.js

```diff
@@ -0,0 +1,58 @@
+import { LitElement } from 'lit';
+
+/**
+ * A component to customize the empty state parameters for a particular filter-dimension-set.
+ * This component does not render anything, but instead gathers data needed for the d2l-filter.
+ */
+class FilterDimensionSetEmptyState extends LitElement {
+
+	static get properties() {
+		return {
+			/**
+			 * The href that will be used for the empty state action. When set with action-text, d2l-filter will render a link action.
+			 * @type {string}
+			 */
+			actionHref: { type: String, attribute: 'action-href' },
```

Comment:
Do you know if Wave plans to use both the action button and the action href, or just one? Just wondering if we can delay adding the second type until someone really needed it, if we thought one was more suited than the other.

Following that same thought - would the href be a bit jarring, to click something inside a filter and have the page navigate?  Maybe a @geurts question

Reply:
That's a good question that I'll forward along. We could start with just one but one of the big things we wanted to emphasize when creating the empty state components was that we wanted people to use the semantically correct action for their use case. We didn't want people to end up using buttons as links.

If we introduce buttons first, then I'm a bit worried people will just use the buttons as links rather than come to us and request that we add functionality for the link action.

Reply:
Chatted with Jeff about this. There are no concerns with navigating with an href inside of a filter. I think it would be a good idea to keep both actions for the above reason.